### PR TITLE
Clear staging hr after tests

### PR DESCRIPTION
**Pull request by hannah38**
Section for uninstalling hrs.
Call that section if on master and aat after tests.

Tested on Draft store
[2020-09-30T07:01:36.046Z] + helm uninstall draft-store-service-staging --namespace rpe
https://build.platform.hmcts.net/blue/organizations/jenkins/HMCTS_Platform%2Fdraft-store/detail/master/133/pipeline/505

### CODE REVIEW

File: vars/sectionDeployToAKS.groovy
Comment (timja) [inline vars/sectionDeployToAKS.groovy:147]: AKS is unneeded, we should go through and remove from all, this is debatable whether it should have it's own stage but because of how the code is structured it's probably easier
```suggestion
              stageWithAgent("Cleanup staging release - $(environment", product) {
```

File: vars/sectionDeployToAKS.groovy
Comment (timja) [inline vars/sectionDeployToAKS.groovy:149]: what about this?
```suggestion
                  helmUninstall(dockerImage, params)
```

File: vars/helmClear.groovy
Comment (timja) [inline vars/helmClear.groovy:21]: I don't understand why this is here? it doesn't appear to be doing anything

File: vars/sectionDeployToAKS.groovy
Comment (timja) [inline vars/sectionDeployToAKS.groovy:147]: looking good, have you tested it somewhere? i.e link to build?

Comment (adusumillipraveen) [reply vars/sectionDeployToAKS.groovy:147]: You need something like : `new Environment(env).nonProdName`

Comment (hannah38) [reply vars/sectionDeployToAKS.groovy]: Hey, I've fixed that and added the links for the test builds

File: vars/helmUninstall.groovy
Comment (adusumillipraveen) [inline vars/helmUninstall.groovy:12]: How about ?
```suggestion
  def aksServiceName = dockerImage.getAksServiceName()
  def namespace = env.TEAM_NAMESPACE

  def chartName = "${params.product}-${params.component}"

  def kubectl = new Kubectl(this, params.subscription, aksServiceName, 
params.aksSubscription.name)
kubectl.login()
```

**Review by adusumillipraveen — CHANGES_REQUESTED** [review]

**Review by adusumillipraveen — DISMISSED** [review]
LGTM, please test once on prod Jenkins before you merge. 👍

**Review by adusumillipraveen — APPROVED** [review]

**Pull request merged**
